docs/code-snippets/outlook-snippets.yaml

```diff
@@ -54,6 +54,22 @@ Office.AppointmentCompose.addItemAttachmentAsync:
 
         Office.context.mailbox.item.addItemAttachmentAsync(itemId, "My Attachment", options, callback);
     }
+Office.AppointmentCompose.body:
+  - |-
+    // This example gets the body of the message in plain text.
+    Office.context.mailbox.item.body.getAsync(
+        "text",
+        { asyncContext:"This is passed to the callback" },
+        function callback(result) {
+            // Do something with the result
+        });
+
+    //The following is an example of the result parameter passed to the callback function.
```

Comment:
I found this to be a bit confusing. Consider something like "The following is an example of an object that is passed as the result parameter to the callback function."